SPECS-EXTENDED/rust-cbindgen/rust-cbindgen.spec

```diff
@@ -0,0 +1,243 @@
+# Generated by rust2rpm 24
+# * tests are completely broken with Cython 3.0.0 on Fedora 39+
+Summary:        Tool for generating C bindings to Rust code
+Name:           rust-cbindgen
+Version:        0.24.3
+Release:        1%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/mozilla/cbindgen
+Source:         %{url}/archive/refs/tags/v0.24.3.tar.gz
+# Below is a manually created tarball, no download link.
+# Note: the cbindgen-%%{version}-cargo.tar.gz file contains a cache created by capturing the contents downloaded into $CARGO_HOME.
+# To update the cache and config.toml run:
+#   tar -xf %{name}-%{version}.tar.gz
+#   cargo vendor > config.toml
+#   tar -czf %{name}-%{version}-cargo.tar.gz vendor/
+#
+Source1:        cbindgen-%{version}-cargo.tar.gz
+Source2:        cargo_config
+%global rustflags '-Clink-arg=-Wl,-z,relro,-z,now'
+%global _description %{expand:
+A tool for generating C bindings to Rust code.}
+BuildRequires:  cargo >= 1.45
+BuildRequires:  rust >= 1.45
+
+%description %{_description}
+
+%package     -n cbindgen
+Summary:        %{summary}
+
+%description -n cbindgen %{_description}
+
+%files -n cbindgen
+%license LICENSE
+%doc README.md
+%doc contributing.md
+%doc docs.md
+%doc internals.md
+%{_bindir}/cbindgen
+
+%package        devel
+Summary:        %{summary}
+BuildArch:      noarch
+
+%description    devel %{_description}
+This package contains library source intended for building other packages which
+use the "cbindgen" crate.
+
+%files devel
+%license LICENSE
+%doc README.md
+%doc contributing.md
+%doc docs.md
+%doc internals.md
+
+%package     -n %{name}-default-devel
+Summary:        %{summary}
+BuildArch:      noarch
+
+%description -n %{name}-default-devel %{_description}
+
+This package contains library source intended for building other packages which
+use the "default" feature of the "cbindgen" crate.
+
+%files -n %{name}-default-devel
+%ghost /Cargo.toml
+
+%package     -n %{name}-clap-devel
+Summary:        %{summary}
+BuildArch:      noarch
+
+%description -n %{name}-clap-devel %{_description}
+
+This package contains library source intended for building other packages which
+use the "clap" feature of the "cbindgen" crate.
+
+%files -n %{name}-clap-devel
+%ghost /Cargo.toml
+
+%prep
+%autosetup -n cbindgen-%{version_no_tilde} -p1
+tar -xf %{SOURCE1}
+install -D %{SOURCE2} .cargo/config
+%if %{with check}
+echo '%{_bindir}/cython'
+echo '%{_bindir}/g++'
+echo '%{_bindir}/gcc'
+%endif
+
+%build
+RUSTFLAGS=%{rustflags} cargo build --release
+
+%install
+export RUSTFLAGS=%{rustflags}
+cargo install --path . --root=%{buildroot}%{_prefix}
+# remove residue crate file
+rm -f %{buildroot}%{_prefix}/.crates*
+%if %{with check}
```

Comment:
lets drop the conditional; we have `with_check` which gets triggered for the test pipeline.